clients/op-geth/Dockerfile

```diff
@@ -1,11 +1,16 @@
 ARG branch=optimism
 FROM us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:$branch
 
-RUN apk add curl jq bash
+RUN apk add curl jq bash tar
 RUN apk add --no-cache ca-certificates
 RUN geth console --exec 'console.log(admin.nodeInfo.name)' --maxpeers=0 --nodiscover --dev 2>/dev/null | head -1 > /version.txt
 
 ADD entrypoint.sh /entrypoint.sh
 RUN chmod +x /entrypoint.sh
 
+# Docker automatically extracts archives when adding them to the image.
+ADD https://storage.googleapis.com/oplabs-goerli-data/goerli-bedrock.tar /goerli-bedrock.tar
+RUN mkdir /goerli-bedrock
+RUN tar -xvf /goerli-bedrock.tar -C /goerli-bedrock
```

Comment:
This doesn't actually get cached between different jobs that build the op-geth client. So we're spending 4 minutes building `op-geth` client docker for each sim (p2p, rpc, l1ops and daisy-chain) even though it's actually only required for daisy-chain. That's quite a bit more data than we actually need to be downloading.

Given that the daisy-chain tests only start up one chain which is shared for all tests, it's probably better to leave this out of the docker image and pass the URL to download the existing db from via an env var instead of just a boolean and download on demand.  Slower for local development I guess, but you could potentially point at a local copy of the tar then.

I'm not strongly opinionated on this though.

Reply:
Swapped the fetching of the bedrock datadir into the entrypoint script so it only happens in the daisy chain task- looks to have sped it up a decent amount.